tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -112,6 +112,40 @@ def test_ebs_single_empty(scheduler, pcluster_config_reader, clusters_factory):
     _test_ebs_correctly_shared(remote_command_executor, mount_dir, scheduler_commands)
 
 
+@pytest.mark.dimensions("ap-northeast-2", "c5.xlarge", "alinux2", "sge")
+@pytest.mark.usefixtures("os", "instance")
+def test_ebs_existing(
+    request, vpc_stacks, region, scheduler, pcluster_config_reader, clusters_factory, snapshots_factory
+):
+    logging.info("Testing ebs existing")
+    existing_mount_dir = "existing_mount_dir"
+
+    logging.info("Creating volume id")
+
+    volume_id = snapshots_factory.create_existing_volume(
+        request, vpc_stacks[region].cfn_outputs["PublicSubnetId"], region
+    )
```

Comment:
it doesn't look like this volume is deleted. can you do it via a fixture like the snapshots_factory fixture defined in test_ebs.py?

Reply:
It is deleted because I am using the snapshots_factory fixture here to create the volume